### PR TITLE
[#3058] improvement(client-python): Correct type specification of create_fileset function

### PR DESCRIPTION
**Pull request by noidname01**
### What changes were proposed in this pull request?

* Change the suggest type of `type` from `Catalog.Type` to `Fileset.Type`

### Why are the changes needed?

Fix: #3508 

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

`./gradlew :clients:client-python:test`

### CODE REVIEW

**Comment by jerryshao** [comment]
@noidname01 can you please add a UT for this?

**Comment by noidname01** [comment]
@jerryshao Sure, working on it

File: clients/client-python/gravitino/catalog/fileset_catalog.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def create_fileset(
         self,
         ident: NameIdentifier,
         comment: str,
-        type: Catalog.Type,
+        type: Fileset.Type,
```
Comment (xunliu) [inline clients/client-python/gravitino/catalog/fileset_catalog.py:95]: Yes, you are right.

The corresponding java code in the here: https://github.com/datastrato/gravitino/blob/main/api/src/main/java/com/datastrato/gravitino/file/FilesetCatalog.java#L76

**Review by xunliu — APPROVED** [review]
LGTM

@noidname01 Thank you fix this issue.

**Pull request merged**